helpdesk_mgmt_timesheet/__manifest__.py

```diff
@@ -20,6 +20,7 @@
         "helpdesk_mgmt_project",
         "hr_timesheet",
         "project_timesheet_time_control",
+        "hr_timesheet_time_type",
```

Comment:
This must be a separate module adding such feature, not adding dependencies to the base module.